chainer/functions/activation/relu.py

```diff
@@ -17,36 +18,62 @@ class ReLU(function_node.FunctionNode):
     """Rectified Linear Unit."""
 
     _use_cudnn = False
+    _use_ideep = False
 
     def check_type_forward(self, in_types):
         type_check.expect(
             in_types.size() == 1,
             in_types[0].dtype.kind == 'f',
         )
 
-    def forward_cpu(self, x):
+    def forward_cpu(self, inputs):
+        if (intel64.should_use_ideep('>=auto')
+                and intel64.inputs_all_ready(inputs)):
+
+            # iDeep implementation
+            # TODO(iDeep): Support arbitrary dimension
```

Comment:
Is this a TODO by us, or iDeep limitation?
(`inputs_all_ready` says `ideep supports array dimension in either 2 or 4 only.`)

In the latter case, maybe we don't need a TODO line in our code.

Reply:
This comment was present in Intel's original implementation.
I think it's obsolete, so I removed it.